projects/vdk-plugins/vdk-notebook/src/vdk/plugin/notebook/notebook_plugin.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import List
+
+from vdk.api.plugin.hook_markers import hookimpl
+from vdk.api.plugin.plugin_registry import IPluginRegistry
+from vdk.internal.core.config import ConfigurationBuilder
+from vdk.internal.core.context import CoreContext
+
+"""
+Include the plugins implementation. For example:
+"""
+
+
```

Comment:
maybe provide a meaningful plugin class name and plugin registry key?

Reply:
I will be adding them in  the next PR, when adding the hook